package.json

```diff
@@ -46,7 +46,7 @@
     "ethereumjs-util": "^6.0.0",
     "fast-memoize": "^2.5.1",
     "merkletreejs": "0.0.22",
-    "openzeppelin-solidity": "2.1.1",
+    "openzeppelin-solidity": "OpenZeppelin/openzeppelin-contracts#master",
```

Comment:
If you check one commit behind, you'll see that neither of these options are working.

Reply:
We don't seem to be running npm install on travis for some weird reason. I think that's why it is failing.

Also, is SafeCast included in a branch or tagged version (e.g. v2.5rc or such). Then we could make it explicit that we are waiting on a specific version.

Reply:
> We don't seem to be running npm install on travis for some weird reason.

I thought this might be the case, but second guessed myself, cuz I thought "How could anything be working if we don't run npm install?"

> Is SafeCast included in a branch or tagged version

I'll check it out.

Reply:
Unfortunately, at this time, there are no branch tags on OZ since their most recent release (which did not yet include these changes)

https://github.com/OpenZeppelin/openzeppelin-contracts/tags

Reply:
Can you use a commit hash? Maybe the commit hash where the SafeCast library was merged

Reply:
I think its just:
```
 "openzeppelin-solidity": "OpenZeppelin/openzeppelin-contracts#2c11ed59faaac4709ab1dc18244dbad3970609e6",
```

Reply:
@nlordell - I like the idea of using the shorter hash. Do we know if this works?

Reply:
@bh2smith absolutely none, I just imagine it would as it probably uses git under the hood to resolve refs

Reply:
How does one know the exact number of characters that constitute a legit "short-hash"?

Reply:
This works, thanks @nlordell - 

"OpenZeppelin/openzeppelin-contracts#2c11ed59"

Reply:
Also, there is no need for `npm install` in the travis file as the `install` script section was not overloaded. Note that travis calls `npm ci` instead of `npm install` which is usually better but uses your `project-lock.json` file - which explains why `SafeCast.sol` was missing. What surprises me is that `npm ci` should fail if `package-lock.json` and `package.json` don't match but didn't in this case.